.github/workflows/tests.yml

```diff
@@ -5,11 +5,12 @@ on: [push, pull_request]
 jobs:
   test:
     name: Build and test
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.runner }}
     strategy:
       fail-fast: false
       matrix:
         version: ["3.8", "3.10", "3.11", "3.12"]
+        runner: ["X64", "ARM64"]
```

Comment:
Is `X64` the same as `amd64`? If so, it would be nice to print that (`amd64`) out instead of `X64`.

Reply:
this is a GH label that only the runners' admin can change. by default GH uses x64. 

it feels unnecessary to add more complexity to the workflow just to translate that into amd64

Reply:
Alright.